### ISSUE TITLE
caching issues - etc

### ISSUE THREAD

**Issue by oojacoboo** [cryptonomex/graphene-ui#865]
Not sure where to start on this, but I can't use the app at all.

Firstly, I clear the cache and it keeps the username and details in tact as if I'm still logged into the app.  I get a notice about having to create a backup, but clicking the backup triggers an error in the console.

I've tried creating new accounts that don't actually create new accounts accessible in the UI, keeps the old wallet registered for a backup, etc.

Honestly, it feels like the buggiest site I've used in a long time.  I really don't have a clue where to start.  I guess the first thing is this cache.  What is up with that?  Where is this data being cached?

**Comment by svk31** [comment]
Firstly, why do you want to clear the cache? Depending on how deep of a clear you're doing you risk deleting your entire wallet.

All the wallet data is kept in an IndexedDB. There's also a fair amount of user settings being stored in localStorage.

Are you using Safary by any chance? If so you can't make backups cause Safari doesn't support the file API properly, you need to use Chrome.

**Comment by oojacoboo** [comment]
Well, at this point, I'm willing to do that.  I can't even use it - period.  It's completely unusable in the current state.  So, starting fresh may allow me to be able to at least use it - even if I have to lose my wallet account, etc.

I'm using Firefox.

**Comment by svk31** [comment]
I'm still not sure what your issue is though to be honest, and how it got in this state you're describing.

There's no login btw, you only unlock your wallet to make transactions and decrypt memos. Apart from that there is no visual difference between a "logged in" and "logged out" state.

**Comment by oojacoboo** [comment]
Right, I understand that, and I'm with you on how it got to this state.  My guess is that it happened b/c I signed up over a year or so ago and haven't really used it to date.

I think the best question here is how do I clear _all_ the cache and start fresh?  I've now wasted 3 usernames in an attempt to get signed up and they're all inaccessible now.  The backup functionality refers to a wallet that's not even associated with a new account created.
